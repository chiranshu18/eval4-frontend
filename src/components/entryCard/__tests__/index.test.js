import EntryCard from '../index';
import { render , screen} from '@testing-library/react';
import React from 'react';

describe('Entries', () => {
  it('Should render correctly', () => {
    const {asFragment} = render(<EntryCard />);
    expect(asFragment()).toMatchSnapshot();
  });
});