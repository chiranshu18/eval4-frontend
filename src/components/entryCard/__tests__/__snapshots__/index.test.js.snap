// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Entries Should render correctly 1`] = `
<DocumentFragment>
  <div
    class="each-entry-card"
  >
    <div
      class="entry-id"
    >
      <p>
        1
      </p>
    </div>
    <div
      class="entry-name"
    >
      <p>
        name
      </p>
    </div>
    <div
      class="entry-website"
    >
      <p>
        text
      </p>
    </div>
    <div
      class="entry-contact"
    >
      <p>
        text
      </p>
    </div>
    <div
      class="entry-actions"
    >
      <img
        alt="delete"
        src="icon1.png"
      />
      <img
        alt="search"
        src="icon1.png"
      />
    </div>
  </div>
  <div
    class="each-entry-card"
  >
    <div
      class="entry-id"
    >
      <p>
        1
      </p>
    </div>
    <div
      class="entry-name"
    >
      <p>
        name
      </p>
    </div>
    <div
      class="entry-website"
    >
      <p>
        text
      </p>
    </div>
    <div
      class="entry-contact"
    >
      <p>
        text
      </p>
    </div>
    <div
      class="entry-actions"
    >
      <img
        alt="delete"
        src="icon1.png"
      />
      <img
        alt="search"
        src="icon1.png"
      />
    </div>
  </div>
  <div
    class="each-entry-card"
  >
    <div
      class="entry-id"
    >
      <p>
        1
      </p>
    </div>
    <div
      class="entry-name"
    >
      <p>
        name
      </p>
    </div>
    <div
      class="entry-website"
    >
      <p>
        text
      </p>
    </div>
    <div
      class="entry-contact"
    >
      <p>
        text
      </p>
    </div>
    <div
      class="entry-actions"
    >
      <img
        alt="delete"
        src="icon1.png"
      />
      <img
        alt="search"
        src="icon1.png"
      />
    </div>
  </div>
  <div
    class="each-entry-card"
  >
    <div
      class="entry-id"
    >
      <p>
        1
      </p>
    </div>
    <div
      class="entry-name"
    >
      <p>
        name
      </p>
    </div>
    <div
      class="entry-website"
    >
      <p>
        text
      </p>
    </div>
    <div
      class="entry-contact"
    >
      <p>
        text
      </p>
    </div>
    <div
      class="entry-actions"
    >
      <img
        alt="delete"
        src="icon1.png"
      />
      <img
        alt="search"
        src="icon1.png"
      />
    </div>
  </div>
  <div
    class="each-entry-card"
  >
    <div
      class="entry-id"
    >
      <p>
        1
      </p>
    </div>
    <div
      class="entry-name"
    >
      <p>
        name
      </p>
    </div>
    <div
      class="entry-website"
    >
      <p>
        text
      </p>
    </div>
    <div
      class="entry-contact"
    >
      <p>
        text
      </p>
    </div>
    <div
      class="entry-actions"
    >
      <img
        alt="delete"
        src="icon1.png"
      />
      <img
        alt="search"
        src="icon1.png"
      />
    </div>
  </div>
  <div
    class="each-entry-card"
  >
    <div
      class="entry-id"
    >
      <p>
        1
      </p>
    </div>
    <div
      class="entry-name"
    >
      <p>
        name
      </p>
    </div>
    <div
      class="entry-website"
    >
      <p>
        text
      </p>
    </div>
    <div
      class="entry-contact"
    >
      <p>
        text
      </p>
    </div>
    <div
      class="entry-actions"
    >
      <img
        alt="delete"
        src="icon1.png"
      />
      <img
        alt="search"
        src="icon1.png"
      />
    </div>
  </div>
  <div
    class="each-entry-card"
  >
    <div
      class="entry-id"
    >
      <p>
        1
      </p>
    </div>
    <div
      class="entry-name"
    >
      <p>
        name
      </p>
    </div>
    <div
      class="entry-website"
    >
      <p>
        text
      </p>
    </div>
    <div
      class="entry-contact"
    >
      <p>
        text
      </p>
    </div>
    <div
      class="entry-actions"
    >
      <img
        alt="delete"
        src="icon1.png"
      />
      <img
        alt="search"
        src="icon1.png"
      />
    </div>
  </div>
</DocumentFragment>
`;
